<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lib_msconfig</class>
 <widget class="QWidget" name="lib_msconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lib_msconfig{background-color: rgb(248, 248, 255);}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>500</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
	font: 14pt &quot;黑体&quot;;
}</string>
   </property>
   <property name="title">
    <string>启动项</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>500</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
font: 12pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QListWidget::item{/*子项样式*/
height:35px;/*高度*/
}
QListWidget::Item:selected {/*选中项样式*/
background-color:rgb(51, 153, 255);
color: rgb(255, 255, 255);
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>38</y>
      <width>500</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="but_add">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>90</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/insert2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_subtract">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/delete2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_save">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/prefix2/LCD_CS351/logo/saves.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_edit">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/prefix2/LCD_CS351/logo/check.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_cancel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/prefix2/LCD_CS351/logo/cancel.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
